docs/codeql/ql-language-reference/ql-language-specification.rst

```diff
@@ -2046,6 +2046,8 @@ A valid stratification must include each predicate and type in the QL program th
 
 A valid stratification must not include the same predicate in multiple layers.
 
+Each non-abstract predicate has an associated body. For predicates inside *declared modules*, this is the predicate declaration. The body of an *instantiation-nested* predicate is the body of the *underlying nested* predicate where all references have been substituted with the *instantiation-relative* entity or alias.
```

Comment:
I think calls need to be added as they aren't a reference in the spec.

Reply:
I have force-pushed a new commit that explicitly mentions calls.